### ISSUE TITLE
Proposal to convert the remaining unminted NCR into CDFT

### ISSUE THREAD

**Issue by Toxic-Cookie** [Neos-Metaverse/NeosPublic#3460]
### What topic do you want to discuss?

The topic I’d like to discuss is the possibility of burning the remaining amount of unminted NCR and potentially repurposing it instead of sending it to the void.

### Goal of discussion

The goal of this discussion is to see what the potential implications of converting the remaining unminted NCR into CDFT would be.

### Context
Using some rough math, as of Dec 15, 2021, the Neos team has accumulated roughly 60 million USD worth of crypto assets. This is enough funding to last for at least a decade give or take. In order to complete the ICO, about 9.5 million NCR would need to be minted. Assuming a pod of whales or an astounding giga comes along and does it all in one shot at the price of ~$10.27, it would take approximately 98 million USD worth of ETH. However, it would be safer to assume that this wouldn’t happen and it would instead vaguely follow the minting curve. In that case, several orders of magnitude more money would be needed to complete the ICO which would result in a ludicrous amount of money I don’t even want to think about.

### CDFT
The current CDFT batch is at 16 of 20 which means that of the total 2 million CDFT created, 80% has already been distributed. This has major implications for the future of the platform. Neos is still in its infancy and has an astronomical amount of room to develop and grow. Incentivizing current and future creators to provide content for the platform is the nutrients needed for that growth. By converting the remaining unminted NCR into CDFT and carefully distributing it to content creators, it benefits every party involved.

### Conclusion
If this proposal were to be put into action, I would imagine that the last four batches of CDFT would be fully distributed and then a new plan to distribute the new CDFT to current and future content creators would be created. That new plan would need an entire discussion of its own. In conclusion, the possibility of converting the remaining unminted NCR into CDFT is a proposal I’d like to thoroughly explore and possibly see become a reality.

**Comment by bluestarpin** [comment]
yes, i agree, with CDFT, we can incentive the creator, especially the VR KOL to creat content in Neos, and with the content existed, the content economy cycle will boost using NCR.

**Comment by alexderpyfox** [comment]
Yes do this

**Comment by CyberWarlock-Work** [comment]
interesting take

**Comment by kulzae** [comment]
Your assuming that 80% of the CDFT has been distributed but in actually it has been minted. Unallocated CDFT funds are held onto as NCR untill a CDFT allocation is granted to someone so the assumption they have already given away 80% of it is incorrect
